src/tests/specs/collaboratorTest.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { browser } from 'protractor';
+import { PlannerPage } from '../page_objects/planner';
+import * as support from '../support';
+
+
+describe('Planner Collaborator Tests:', () => {
+  let planner: PlannerPage;
+  let planner1: PlannerPage;
+  let c = new support.Constants();
+
+  beforeAll(async () => {
+    await support.desktopTestSetup();
+    planner = new PlannerPage(browser.baseUrl);
+    await planner.openInBrowser();
+    // // This is necessary since the planner takes time to load on prod/prod-preview
+    await browser.sleep(5000);
+    await planner.ready();
+    let url = await browser.getCurrentUrl()
+    let urlPathName = await browser.executeScript('return document.location.pathname');
+    browser.baseUrl = url.replace(urlPathName,'/rbajpai-test-preview/DO_NOT_DELETE/plan');
+    planner1 = new PlannerPage(browser.baseUrl);
```

Comment:
You can use `planner = new PlannerPage(...)` . 
We won't be using the old value of planner, right?

Reply:
No